server/game/cards/02.2-TRtW/SyrioForel.js

```diff
@@ -4,42 +4,23 @@ class SyrioForel extends DrawCard {
     setupCardAbilities(ability) {
         this.action({
             title: 'Give a character military icon and stealth',
-            method: 'giveIcon',
-            limit: ability.limit.perPhase(1)
+            limit: ability.limit.perPhase(1),
+            phase: 'challenge',
+            target: {
+                cardCondition: card => card.getType() === 'character' && card.controller === this.controller
```

Comment:
Syrio can be used on opponent's character, so the controller check here should be removed